Userland/Applications/PixelPaint/Image.cpp

```diff
@@ -409,13 +409,13 @@ void Image::merge_active_layer_up(Layer& layer)
 
 void Image::merge_active_layer_down(Layer& layer)
 {
-    if (m_layers.size() < 2)
-        return;
+    // NOTE: The action for this should be disabled if there are fewer than 2 layers.
+    VERIFY(m_layers.size() >= 2);
+
     int layer_index = this->index_of(layer);
-    if (layer_index == 0) {
-        dbgln("Cannot merge layer down: layer is already at the bottom");
-        return; // FIXME: Notify user of error properly.
-    }
+
+    // NOTE: The action for this should be disabled if the active layer is already bottommost.
```

Comment:
At least you are consistent with line 400 here. top / bottom are proper opposites, but front / back is used in most of the other contexts here.